tests/server/models/relations-test.js

```diff
@@ -7,259 +7,216 @@ var Order = require('../../../server/db/models/order');
-                content: 'the worst evilest robot i ever saw'});
-      var product=Product.create({ name: 'Unicorn robot', description: 'Most beautiful creature', inventory: 10, currentPrice: 100.00});
 
-      return Promise.all([product,review1,review2])
-      .spread(function(product,review1,review2){
-        reviewsArr=[review1,review2]
-        return product.addReviews(reviewsArr);
-      })
+  describe('Product',function(){
+    it('belongs to many categories',function(){
+      return productsArr[0].addCategories(categoriesArr)
       .then(function(){
         return Product.findOne({
               where: { name: 'Unicorn robot'},
-              include: { model: Review}
+              include: { model: Category}
             })
       })
       .then(function(product){
-        return product.getReviews();
+        expect(product.categories[0].name).to.equal('Leisure');
+        expect(product.categories[1].name).to.equal('Evil');
       })
-      .then(function(reviews){
-        var reviewContents=[]
-        reviews.forEach(function(review){
-          reviewContents.push(review.dataValues.content)
+    })
+    it('has many reviews',function(){
+        return productsArr[0].addReviews(reviewsArr)
+        .then(function(product){
+          return product.getReviews();
         })
-        expect(reviewContents).to.contain('the best damn unicorn i ever rode');
-        expect(reviewContents).to.contain('the worst evilest robot i ever saw');
+        .then(function(reviews){
+          var reviewContents=[]
+          reviews.forEach(function(review){
+            reviewContents.push(review.dataValues.content)
+          })
+          expect(reviewContents).to.contain('the best damn unicorn i ever rode');
+          expect(reviewContents).to.contain('the worst evilest robot i ever saw');
 
+        })
       })
-    })
-})
+  })
 
-describe('Review',function(){
-  beforeEach(function () {
-    return db.sync({force: true});
-  });
-  it('belongs to a product',function(){
-    var review1=Review.create({stars: 5,content: 'the best damn unicorn i ever rode'});
-    var product1=Product.create({ name: 'Unicorn robot', description: 'Most beautiful creature', inventory: 10, currentPrice: 100.00});
-    return Promise.all([review1,product1])
-    .spread(function(review,product){
-      return review.setProduct(product);
-    })
-    .then(function(review){
-      return Review.findById(review.dataValues.id,{include: {model: Product}})
-    })
-    .then(function(review){
-      return review.getProduct();
+  describe('Review',function(){
+    it('belongs to a product',function(){
+      return reviewsArr[0].setProduct(productsArr[0])
+      .then(function(review){
+        return Review.findById(review.dataValues.id,{include: {model: Product}})
+      })
+      .then(function(review){
+        return review.getProduct();
+      })
+      .then(function(product){
+        expect(product.dataValues.name).to.equal('Unicorn robot');
```

Comment:
You shouldn't have to use `dataValues`. You should be able to access `product.name`.